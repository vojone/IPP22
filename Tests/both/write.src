.ippcode22

WRITE string@a\032b\010c

WRITE int@1

WRITE nil@nil

WRITE bool@false

DEFVAR GF@b

MOVE GF@b int@5
WRITE GF@b

MOVE GF@b nil@nil
WRITE GF@b

MOVE GF@b string@string
WRITE GF@b