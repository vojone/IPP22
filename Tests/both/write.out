a b
c1false5string